src/pretalx/submission/models/submission.py

```diff
@@ -423,6 +447,14 @@ def slot(self):
             else None
         )
 
+    @property
+    def slots(self):
```

Comment:
Please absolutely do not name this method ``slots``, als this property already exists and references *all* talk slots related to this submission. ``current_slots`` might be appropriate.

Reply:
as it is not used currently i remove it.